applications/rag/frontend/README.md

```diff
@@ -3,98 +3,21 @@
 This directory contains the code for a frontend flask webserver integrating with the inference
 backend and (NYI) vector database through [LangChain](https://python.langchain.com/docs/get_started/introduction).
 
-## Cloud Run Setup
+The image is hosted on `us-central1-docker.pkg.dev/ai-on-gke/rag-on-gke` and used by the frontend deployment.
 
-1. Configure inference internal loadbalancer service:
-```
-export INFERENCE_DEPLOYMENT=mistral-7b-instruct  # Customize as needed
-kubectl create service loadbalancer $INFERENCE_DEPLOYMENT --tcp=80:8080 && \
-kubectl annotate service/$INFERENCE_DEPLOYMENT "cloud.google.com/load-balancer-type=Internal"
+## Sensitive Data Protection
+Data Loss Prevention (DLP) and Natural Language Protection (NLP) System. Our integrated solution leverages the latest in data protection and language understanding technologies to safeguard sensitive information and ensure content appropriateness across digital platforms.
```

Comment:
Let's refer to Cloud Natural Language API instead and link through to it. NLP as listed here isn't a product and isn't a part of SDP. Link to DLP and SDP pages so people can learn more.